### ISSUE TITLE
FITSFigure.set_nan_color not working

### ISSUE THREAD

**Issue by ttshimiz** [aplpy/aplpy#418]
I'm not sure if this is related to #402 but FITSFigure.set_nan_color does not seem to working for aplpy v2.0 with matplotlib v3.0.2. Here is a MWE:

```python
import aplpy
import astropy.io.fits as fits
import numpy as np
import matplotlib.pyplot as plt

img = np.random.random((10, 10))
img[5, :] = np.nan
hdu = fits.PrimaryHDU(data=img)
fig = aplpy.FITSFigures(hdu)

fig.show_colorscale()
fig.set_nan_color('w')
```
Nothing seems to happen after trying to set the nan color. However if I just do a normal pyplot.imshow and change the nan color using set_bad then it works just fine so I don't think its a Matplotlib issue.

```python
ax = plt.imshow(img, origin='lower', cmap='viridis')
cm = ax.get_cmap()
cm.set_bad('w')
ax.set_cmap(cm)
```

**Comment by astrofrog** [comment]
Just confirming this is a bug, I'll have a fix out soon.

**Comment by astrofrog** [comment]
@ttshimiz - this is fixed in APLpy 2.0.2

**Comment by ttshimiz** [comment]
Thanks @astrofrog!
